### PR TITLE
feat: add support listStyleType by alphabet

### PR DESCRIPTION
**Pull request by robyhuzwandar**

### CODE REVIEW

File: src/renderers.tsx [hunk 1]
```diff
@@ -86,6 +87,8 @@ const renderers: HtmlRenderers = {
       listStyle.listStyle ||
       '';
 
+    console.log('ROBY LOG : ', listStyleType.includes('lower-alpha'));
```
Comment (danomatic) [inline src/renderers.tsx:90]: Can you remove the console.log?

**Review by danomatic — CHANGES_REQUESTED** [review]
Looks great other than the console.log!

File: src/renderers.tsx (hunk 1 above)
Comment (robyhuzwandar) [reply src/renderers.tsx]: removed

**Comment by robyhuzwandar** [comment]
when I can get an update on the npm? @danomatic

**Pull request merged**